include/revng/Model/CallEdge.h

```diff
@@ -52,12 +52,18 @@ class model::CallEdge : public model::generated::CallEdge {
   CallEdge() : model::generated::CallEdge() { Type = AssociatedType; }
 
   CallEdge(MetaAddress Destination, FunctionEdgeType::Values Type) :
-    model::generated::CallEdge(Destination, Type) {}
+    model::generated::CallEdge(model::FunctionEdgeBaseKind::CallEdge,
```

Comment:
I would expect the Kind to be automatically provided by the generated constructor base class, can't we do that?